build.gradle

```diff
@@ -362,6 +414,19 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
 
     compileOnly "org.opensearch:opensearch:${opensearch_version}"
+
+    //integration test framework:
+    integrationTestImplementation('com.carrotsearch.randomizedtesting:randomizedtesting-runner:2.7.1') {
+        exclude(group: 'junit', module: 'junit')
+    }
+    integrationTestImplementation 'junit:junit:4.13.2'
+    integrationTestImplementation "org.opensearch.plugin:reindex-client:${opensearch_version}"
+    integrationTestImplementation "org.opensearch.plugin:percolator-client:${opensearch_version}"
+    integrationTestImplementation 'commons-io:commons-io:2.11.0'
```

Comment:
I'm curious, why do dependencies need to be repeated for `integrationTestImplementation` if we already have an entry for `testImplementation`?

Reply:
As the goal of the project is the replacement of large parts of the old legacy tests, we want to keep the test definitions as separate as possible.